tests/test_project.py

```diff
@@ -229,6 +229,31 @@ def test_project_suggest(registry):
     assert hits[0].score == 1.0
 
 
+def test_project_suggest_transform_limit(registry):
+    project = registry.get_project("limit-transform")
+    result = project.suggest("this is some text")
+    assert len(result) == 0
+
+
+def test_project_suggest_batch(registry, fulltext_corpus):
+    project = registry.get_project("dummy-en")
+    result = project.suggest_batch(fulltext_corpus)
+    assert len(result) == 28  # Number of documents
+    fist_doc_hits = result[0].as_list()
+    assert len(fist_doc_hits) == 1
+    assert fist_doc_hits[0].subject_id == project.subjects.by_uri(
```

Comment:
Typo here: `fist` should probably be `first`..? (Was this copied from somewhere else with the same typo?)